.gitignore

```diff
@@ -18,7 +18,6 @@
 
 config/database.yml
 db/schema.rb
-.rspec
```

Comment:
I prefer to ignore `.rspec`. __for me__ dotfiles are user specific configuration. 

But it is ok :v: